keps/sig-auth/541-external-credential-providers/README.md

```diff
@@ -493,6 +493,114 @@ credentials).
 PEM format. The certificate must be valid at the time of execution. These
 credentials are used for mTLS handshakes.
 
+### Metrics
+
+As discussed [below](#rollout-upgrade-and-rollback-planning), there are 4
+primary metrics used by this feature set.
+
+```golang
+var (
```

Comment:
Is the duration of the exec plugin call a useful metric? I could see it being helpful in debugging long-running API calls, but perhaps it is more effort than it is worth.

Reply:
@ehashman I'd also be interested in your comments here, if you have cycles. Have you seen this type of metric being useful to operators in other scenarios?